src/routes/json.rs

```diff
@@ -13,7 +13,7 @@ pub struct Entry {
     pub text: String,
     pub extension: Option<String>,
     pub expires: Option<u32>,
-    pub burn_after_reading: Option<bool>,
+    pub burn_after_reading: bool,
```

Comment:
Isn't that a breaking change with `burn_after_reading` being a required field now?

Reply:
True, dropped that commit, since the Json entry needs the Option for URL API stability and the database entry needs the Option to not break existing databases after upgrade.